### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by ylaassel**

### CODE REVIEW

File: docs/api/tray.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Sets the hover text for this tray icon.
 
 Sets the title displayed aside of the tray icon in the status bar.
 
-This is only implmented on OS X.
+This is only implemented on OS X.
```
Comment (bwin) [inline docs/api/tray.md]: Not sure if it's worth the extra byte.

Comment (bwin) [inline docs/api/tray.md]: JK

**Comment by bwin** [comment]
Thanks for contributing.

File: docs/api/tray.md (hunk 1 above)
Comment (ylaassel) [inline docs/api/tray.md]: Haha

**Comment by ylaassel** [comment]
No worries. Spotted while reading the docs.

**Comment by kevinsawicki** [comment]
Thanks!

**Pull request merged**